src/aap_eda/settings/default.py

```diff
@@ -11,823 +11,39 @@
-# RULEBOOK LIVENESS SETTINGS
-# ---------------------------------------------------------
-
-RULEBOOK_READINESS_TIMEOUT_SECONDS = int(
-    settings.get("RULEBOOK_READINESS_TIMEOUT_SECONDS", 60)
-)
-RULEBOOK_LIVENESS_CHECK_SECONDS = int(
-    settings.get("RULEBOOK_LIVENESS_CHECK_SECONDS", 300)
-)
-RULEBOOK_LIVENESS_TIMEOUT_SECONDS = (
-    int(settings.get("RULEBOOK_LIVENESS_TIMEOUT_SECONDS", 310))
-    + RULEBOOK_LIVENESS_CHECK_SECONDS
-)
-ACTIVATION_RESTART_SECONDS_ON_COMPLETE = int(
-    settings.get("ACTIVATION_RESTART_SECONDS_ON_COMPLETE", 0)
-)
-ACTIVATION_RESTART_SECONDS_ON_FAILURE = int(
-    settings.get("ACTIVATION_RESTART_SECONDS_ON_FAILURE", 60)
-)
-ACTIVATION_MAX_RESTARTS_ON_FAILURE = int(
-    settings.get("ACTIVATION_MAX_RESTARTS_ON_FAILURE", 5)
-)
-
-# -1 means no limit
-MAX_RUNNING_ACTIVATIONS = int(settings.get("MAX_RUNNING_ACTIVATIONS", 5))
-
-# ---------------------------------------------------------
-# RULEBOOK ENGINE LOG LEVEL
-# ---------------------------------------------------------
-
-
-# For backwards compatibility, from the old value "-v" to the new value "info"
-def get_rulebook_process_log_level() -> RulebookProcessLogLevel:
-    log_level = settings.get(
-        "ANSIBLE_RULEBOOK_LOG_LEVEL",
-        "error",
-    )
-    if log_level is None:
-        return RulebookProcessLogLevel.ERROR
-    if log_level.lower() == "-v":
-        return RulebookProcessLogLevel.INFO
-    if log_level.lower() == "-vv":
-        return RulebookProcessLogLevel.DEBUG
-    if log_level not in RulebookProcessLogLevel.values():
-        raise ImproperlyConfigured(
-            f"Invalid log level '{log_level}' for ANSIBLE_RULEBOOK_LOG_LEVEL"
-            f" setting. Valid values are: {RulebookProcessLogLevel.values()}"
-        )
-    return RulebookProcessLogLevel(log_level)
+from .post_load import post_loading
 
+EDA_SETTINGS_FILE = "/etc/eda/settings.yaml"
 
-ANSIBLE_RULEBOOK_LOG_LEVEL = get_rulebook_process_log_level()
-ANSIBLE_RULEBOOK_FLUSH_AFTER = settings.get(
-    "ANSIBLE_RULEBOOK_FLUSH_AFTER", 100
+DYNACONF = factory(
+    __name__,
+    "EDA",
+    # Options passed directly to dynaconf
+    environments=("production", "development", "testing"),
+    settings_files=["defaults.py"],
 )
 
-# ---------------------------------------------------------
-# DJANGO ANSIBLE BASE SETTINGS
-# ---------------------------------------------------------
-from ansible_base.lib import dynamic_config  # noqa: E402
+load_standard_settings_files(
+    DYNACONF
+)  # /etc/ansible-automation-platform/*.yaml
+DYNACONF.load_file(EDA_SETTINGS_FILE)
+load_envvars(DYNACONF)  # load envvars prefixed with EDA_
+DYNACONF.load_file("constants.py")  # load internal non-overwritable settings
+post_loading(DYNACONF)
+export(__name__, DYNACONF)  # export back to django.conf.settings
 
 dab_settings = os.path.join(
     os.path.dirname(dynamic_config.__file__), "dynamic_settings.py"
 )
 include(dab_settings)
```

Comment:
You can now replace this with:

```suggestion
load_dab_settings(DYNACONF)

export(__name__, DYNACONF)  # export back to django.conf.settings
```

`load_dab_settings` is now implemented as a separate function to allow you to call it later.

Ensure `export` is the last thing you call in the django settings module